admin_guide/node-allocatable-resources.adoc

```diff
@@ -0,0 +1,58 @@
+= Node Allocatable resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node has a chance to specify how much resources get consumed
+by kubernetes related components (e.g. kubelet, kube-proxy, docker) running on a node
```

Comment:
@derekwaynecarr openshift-node is one binary that starts all underlying components (kube-proxy, kubelet, docker, ...). So what about:
```
by all underlying components (e.g. kubelet, kube-proxy, docker) that openshift-node runs on a node
```

@detiber IIUC, openshift-node is a generic term for a set of components/daemons that are run/spawn on each node openshift-node component occupies. It should cover both OSE/AP and Origin. But I may be mistaken.